v22.2/backup-and-restore-monitoring.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Backup and Restore Monitoring
+summary: An overview of the metrics to monitor backup and restore jobs in CockroachdB.
+toc: true
+docs_area: manage
+---
+
+CockroachDB includes metrics to monitor [backup](backup.html), [restore](restore.html), and [scheduled backup](create-schedule-for-backup.html) jobs. You can use monitoring integrations to alert when there are anomalies, such as backups that have failed or restore jobs encountering a retryable error.
+
+Depending on whether you are using a {{ site.data.products.dedicated }} or {{ site.data.products.core }} cluster, you can use the following to monitor backup and restore metrics for your CockroachDB cluster:
+
+- [Prometheus endpoint](#prometheus-endpoint): {{ site.data.products.dedicated }}, {{ site.data.products.core }}
+- [Datadog integration](#datadog-integration): {{ site.data.products.dedicated }}
+
```

Comment:
I wonder if we should specifically call out the schedules_backup_last_completed_time metric as the recommended metric to monitor, and/or mention that whichever schedule -- if you have more than one -- maintains the backups you'd use in a disaster should be configured to update this metric.